service/login.js

```diff
@@ -0,0 +1,29 @@
+let fs = require('fs');
```

Comment:
This file is already in the repository, please pull from master to your current working branch and resubmit PR
i.e.
git checkout addedSearchFeature
git pull origin master